core/build_playerglobal/src/lib.rs

```diff
@@ -726,6 +733,15 @@ fn write_native_table(data: &[u8], out_dir: &Path) -> Result<Vec<u8>, Box<dyn st
             #(#rust_custom_constructors,)*
         ];
 
+        // This is an array containing the method ids of every method marked
+        // "[Ruffle(FastCall)]". Unlike the rest, it is not indexed by method id-
+        // instead, every item in the list is a method id.
+        //
+        // FIXME: should this be some sort of hashset?
```

Comment:
If you sorted the vec before putting it in, we could use `binary_search` instead of `iter().any()` to speed up* lookups without affecting memory use at all.
Alternatively, there's always https://github.com/rust-phf/rust-phf to construct a static hashset at compile time (and thus not waste any allocations), but might be an overkill here.

(*I imagine the difference might not matter as long as we have <100 fast methods. )